carma/launch/guidance.launch

```diff
@@ -108,8 +108,10 @@
   <include file="$(find pure_pursuit_jerk_wrapper)/launch/pure_pursuit_jerk_wrapper.launch" />
 
     <!-- Route Following Plugin -->
-  <include file="$(find route_following_plugin)/launch/route_following_plugin.launch" />
-  
+  <group>
+    <include file="$(find route_following_plugin)/launch/route_following_plugin.launch" />
+    <remap from="upcoming_lane_change_status" to="$(optenv CARMA_GUIDE_NS)/upcoming_lane_change_status"/>
```

Comment:
I believe this remapping will not work. remappings are applied top-down within scope. The remappning is scoped to the group tag but there is nothing below it, so it has no impact on the system.